python/lsst/ts/cbp/config_schema.py

```diff
@@ -100,15 +100,7 @@
         type: number
         description: Rotation of mask (degree)
         default: 150
-  encoder_tolerance:
-    description: Encoder error tolerance for in_position event
-    type: number
-    default: 0.1
-  focus_crosstalk:
-    description: Focus encoder crosstalk when moving tolerance
-    type: number
-    default: 0.4
-required: [address, port, mask1, mask2, mask3, mask4, mask5, encoder_tolerance, focus_crosstalk]
```

Comment:
What's the reasoning behind removing the tolerance values from the configuration? You'll also need to update the version number of the schema if you want to keep this change.

Reply:
The tolerance values shouldn't ever change, like timeouts. So I think they are more appropriate being class variables.

Reply:
To change the version number, do I just do that at the `title` of the schema? Or does it have to happen elsewhere as well?

Reply:
I presume I have to make a `v2` in [ts_config_mtcalsys/CBP](https://github.com/lsst-ts/ts_config_mtcalsys)

Reply:
Just in the title of the schema and yes you are correct.

Reply:
https://github.com/lsst-ts/ts_config_mtcalsys/pull/30